src/pages/[platform]/build-a-backend/storage/upload/index.mdx

```diff
@@ -231,7 +231,7 @@ Upload a file in the browser:
 ```javascript
 import { uploadData } from 'aws-amplify/storage';
 
-const uploadDataInBrowser = async () => {
+const uploadDataInBrowser = async (event) => {
```

Comment:
Can we please test this code quickly in a browser? the reference to `filename` on line 238 is likely going to be undefined

Reply:
fixed

Reply:
@reesscot Its just a sample code right. It is kinda of assumed that customer passes in their file name in there. Are you suggesting we add a comment here or just make it a string?

Reply:
Note: Looking through the docs, that how we reference parameter input everywhere

Reply:
@ashika112 I think the difference here is we are providing what looks like an encapsulated function for handling an browser upload, so we shouldn't be using undefined variables.